contracts/PackageIndex.sol

```diff
@@ -65,15 +65,15 @@ contract PackageIndex is Authorized, PackageIndexInterface {
   /// @param patch The patch portion of the semver version string.
   /// @param preRelease The pre-release portion of the semver version string.  Use empty string if the version string has no pre-release portion.
   /// @param build The build portion of the semver version string.  Use empty string if the version string has no build portion.
-  /// @param releaseLockfileURI The URI for the release lockfile for this release.
+  /// @param releaseManifestURI The URI for the release manifest for this release.
   function release(
     string name,
     uint32 major,
     uint32 minor,
     uint32 patch,
     string preRelease,
     string build,
-    string releaseLockfileURI
+    string releaseManifestURI
```

Comment:
I think this can just be `manifestURI`, because:
a. it used to be called "release lockfile"
b. now it is called "package manifest"
c. the word `manifest` is clear on its own in this context

Reply:
Yes that seems right to me too . . .

Reply:
@gnidan Made this change - there's *one* place where maybe clarity is diminished by this and its also at the EIP layer:
```
- packageIndex.getReleaseManifestURI
+ packageIndex.getManifestURI
```

Do you think it should be excepted from the shortening or it's fine?

Reply:
Oh sorry now I see you were actually pointing at exactly this part! Ignore. . . .